src/qpub/dgaf.code-workspace

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
add this extension to git ignore?

Reply:
ewww how did this get here. thanks for catching this,